tests/system/apmserver.py

```diff
@@ -30,6 +29,10 @@
 diagnostic_interval = float(os.environ.get('DIAGNOSTIC_INTERVAL', 0))
 
 
+def is_subset(s, d):
+    return all(d.get(key, None) == val for key, val in s.items())
```

Comment:
Can we use [set.issubset](https://docs.python.org/3.7/library/stdtypes.html#frozenset.issubset) from stdlib?

Reply:
yes; didn't bother to convert to `set` just for the check (shouldn't make a difference performance wise) but agree it's better to use the existing one.